_blog.yml

```
@@ -5227,3 +5227,15 @@
     - community
     - research
     - open-source-collab
+
+- local: run-comfyui-workflows-on-spaces
+  title: "Run ComfyUI workflows for free with Gradio on Spaces"
```

consider shorter title
```suggestion
  title: "Run ComfyUI workflows for free on Spaces"
```